docs/groovy-script/getting_started/run_config.md

```diff
@@ -0,0 +1,185 @@
+---
+order: 1000
+---
+
+# Run Config
+
+This file defines how each script file should be executed.
+It can also store some general info about the mod pack.
+The file will be generated if it doesn't exist.
+If you don't understand what this is or how it works you can skip this.
+All you need to know is that you put your scripts with recipes in `groovy/postInit`.
+Scripts with stuff like Item Creation go in `groovy/preInit`.
+
+Let's see what the file can look like.
+
+::: info Example {id="example"}
+```json
+{
+  "packName": "",
+  "packId": "",
+  "version": "1.0.0",
+  "debug": false,
+  "classes": [
+    "classes/"
+  ],
+  "loaders": {
+    "preInit": [
+      "preInit/"
+    ],
+    "postInit": [
+      "postInit/"
+    ]
+  },
+  "packmode": {
+    "values": [
+      "normal",
+      "hard"
+    ],
+    "default": "normal",
+    "integratePackmodeMod": false
+  }
+}
+```
+:::
+
+Let's go through it bit by bit:
+
+
+## `packName`
+
+The name of the pack.
+The pack name can be anything. It's the name that will show up in JEI in tooltips on items you created.
+Important for [content](../content/index.md).
+- Can be accessed in a script via `getPackName()`.
+
+## `packId`
+
+The id of the pack.
+- Can be accessed in a script via `getPackId()`.
+The pack id is very important. It must only consist of lower case letters and `_`.
```

Comment:
while it is true that it can be omitted if you only add simple recipes, given that it is required for
1. anything creating content
2. the most basic form of recipe creation (have to explicitly declare the `name` field to avoid it)

i think the description should actually be intensified to indicate just how required it is.

also i do note that changing the id can cause item loss just below this in a Danger block.
```md
::: info Danger {id="danger"}
Changing the pack id will result in created items being lost in existing worlds!
:::
```